### PR TITLE
Refactor renderList method

### PR DESCRIPTION
**Pull request by bazal4o**
Bug 221308

### CODE REVIEW

File: src/js/modules/infragistics.ui.editors.js [hunk 1]
```diff
@@ -22,35 +22,35 @@ if (typeof jQuery !== "function") {
...
 			*/
 			height: null,
-			/* type="object" Gets sets value in editor. The effect of setting/getting that option depends on type of editor and on dataMode options for every type of editor.*/
+			/* type="object" Gets/Sets value in editor. The effect of setting/getting that option depends on type of editor and on dataMode options for every type of editor.*/
 			value: null,
-			/* type="number" Gets sets value in tabIndex for editor. */
+			/* type="number" Gets/Sets value in tabIndex for editor. */
 			tabIndex: null,
-			/* type="bool" Sets gets ability to prevent null value.
+			/* type="bool" Sets/Gets ability to prevent null value.
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.editors.js]: Make this consistent. Follow Gets/Sets.

File: src/js/modules/infragistics.ui.editors.js [hunk 2]
```diff
@@ -22,35 +22,35 @@ if (typeof jQuery !== "function") {
...
 				If that option is false, and editor has no value, then value is set to an empty string.
 			*/
 			allowNullValue: false,
-			/* type="string|number|null" Sets gets the representation of null value. In case of default the value for the input is set to null, which makes the input to hold an empty string */
+			/* type="string|number|null" Sets/Gets the representation of null value. In case of default the value for the input is set to null, which makes the input to hold an empty string */
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.editors.js]: Make this consistent. Follow Gets/Sets

File: src/js/modules/infragistics.ui.editors.js [hunk 3]
```diff
@@ -22,35 +22,35 @@ if (typeof jQuery !== "function") {
...
 			nullValue: null,
 			/* type="string" Sets the name attribute of the value input. This input is used to sent the value to the server. In case the target element is input and it has name attribute, but the developer has set the inputName option, so this option overwrites the value input and removes the attribute from the element. */
 			inputName: null,
 			/* type="bool" Gets/Sets the readonly attribute.Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is sent into the request.*/
 			readOnly: false,
 			/* type="bool" Gets/Sets the disabled attribute.Does not allow editing. Disables all the buttons and iteracitons applied. On submit the current value is not sent into the request*/
 			disabled: false,
-			/* type="object" Sets gets options supported by the igValidator widget.
+			/* type="object" Sets/Gets options supported by the igValidator widget.
```
Comment (kdinev) [inline src/js/modules/infragistics.ui.editors.js]: Make this consistent. Follow Gets/Sets

**Comment by kdinev** [comment]
@bazal4o Also analyze which lines are not covered after the refactor and update the tests.

**Comment by bazal4o** [comment]
image

The refactored method is covered

**Pull request merged**